sdk/servicebus/test-resources.json

```diff
@@ -230,11 +233,11 @@
     },
     "AZURE_SERVICEBUS_FULLY_QUALIFIED_DOMAIN_NAME": {
       "type": "string",
-      "value": "[concat(variables('namespaceName'), '.', variables('namespaceDomainNameSuffix'))]"
```

Comment:
This seems to be the better way to use the domain suffix

Reply:
This branch for internal testing purposes only, so please ignore its.